### PR TITLE
BC21 - Add eox middlewares.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR adds middleware functions that comes from edunext-openedx-extensions.
Middlewares added:
 - RedirectionsMiddleware
 - PathRedirectionMiddleware

## Reviewers:

 - [ ] @felipemontoya 
 - [ ] @diegomillan

### CODE REVIEW

File: eox_tenant/admin.py [hunk 1]
```diff
@@ -67,4 +67,17 @@ def ednx_signal(self, microsite):
             return str(error)
 
 
+class RedirectionAdmin(admin.ModelAdmin):
+    """
+    TODO: add me
```
Comment (felipemontoya) [inline eox_tenant/admin.py]:
```suggestion
    Admin view to see and edit edunext redirection objects
```

File: eox_tenant/management/commands/change_domain.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    help = """This function will iterate over all microsites objects
+            to change microsite prod domains to a stage versions."""
+    suffix_stage_domain = ""
+
+    def add_arguments(self, parser):
+        """
+        Add optional domain from the line command
+        """
+        parser.add_argument('suffix_domain', type=str,
+                            nargs='?', default=u"stage.edunext.co")
```
Comment (felipemontoya) [inline eox_tenant/management/commands/change_domain.py]: Can we read this default form the settings?

**Comment by Squirrel18** [comment]
@felipemontoya Feedback addresed.

File: eox_tenant/settings/common.py [hunk 3]
```diff
@@ -42,3 +42,4 @@ def plugin_settings(settings):
     settings.EOX_MAX_CONFIG_OVERRIDE_SECONDS = 300
     settings.EDXMAKO_MODULE_BACKEND = 'eox_tenant.edxapp_wrapper.backends.edxmako_h_v1'
     settings.UTILS_MODULE_BACKEND = 'eox_tenant.edxapp_wrapper.backends.util_h_v1'
+    settings.CHANGE_DOMAIN_DEFAULT_SITE_NAME = "stage.edunext.co"
```
Comment (felipemontoya) [inline eox_tenant/settings/common.py]: you need to override this in aws

File: eox_tenant/management/commands/change_domain.py [hunk 4]
```diff
@@ -25,8 +26,9 @@ def add_arguments(self, parser):
         """
         Add optional domain from the line command
         """
+        default_name_value = getattr(settings, 'CHANGE_DOMAIN_DEFAULT_SITE_NAME', 'stage.edunext.co')
```
Comment (felipemontoya) [inline eox_tenant/management/commands/change_domain.py]: no need to have the same default here, you can leave ""

**Comment by felipemontoya** [comment]
@Squirrel18 could you please squash the las 4 commits before merging?

**Pull request merged**